### PR TITLE
Feature/300 display keyresults on objective

### PR DESCRIPTION
**Pull request by RandomTannenbaum**
- **Attention**: Branch was created from feature/303-create-overview
- Add keyresult component
- Load data that it gets from objective component with @Input()
- Include keyresult in objective component, so its displayed correctly
- No tests needed

### CODE REVIEW

File: frontend/src/app/keyresult/keyresult.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="key-result w-100 h-100 rounded-2 mb-3 p-3">
+  <div class="lh-base">{{ keyResult.title }}</div>
+  <p *ngIf="keyResult.lastCheckIn.createdOn != undefined; else noCheckin" class="last-checkin">
```
Comment (tuxmatta) [inline frontend/src/app/keyresult/keyresult.component.html:3]: div verwenden

File: frontend/src/app/keyresult/keyresult.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    Letztes Checking: {{ keyResult.lastCheckIn.createdOn | date }}
+  </p>
+  <ng-template #noCheckin class="last-checkin">Noch kein Check-In vorhanden</ng-template>
```
Comment (tuxmatta) [inline frontend/src/app/keyresult/keyresult.component.html:6]: Auch mit einem div

File: frontend/src/app/keyresult/keyresult.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-keyresult',
+  templateUrl: './keyresult.component.html',
+  styleUrls: ['./keyresult.component.scss'],
+})
+export class KeyresultComponent implements OnInit {
+  @Input() keyResult!: KeyresultMin;
+
+  constructor() {}
+
+  ngOnInit(): void {}
```
Comment (tuxmatta) [inline frontend/src/app/keyresult/keyresult.component.ts:14]: ngOnInit() löschen

**Review by tuxmatta — CHANGES_REQUESTED** [review]

File: frontend/src/styles.scss [hunk 4]
```diff
@@ -57,6 +57,10 @@ body {
   word-wrap: break-word;
 }
 
+.sub-title {
+  font-size: 0.7em;
```
Comment (peggimann) [inline frontend/src/styles.scss:61]: use rem

**Review by peggimann — CHANGES_REQUESTED** [review]

**Pull request merged**